types/passport/index.d.ts

```diff
@@ -25,6 +26,9 @@ declare global {
             isAuthenticated(): boolean;
             isUnauthenticated(): boolean;
         }
+        // tslint:disable-next-line:no-empty-interface
+        interface User {
```

Comment:
Can you add documentation that users have to manually augment this? This will technically break existing users who relied on the `any`, so at the least we should give them guidance on how to re-open the `User` interface.

Reply:
you are right!

but it would be not a breaking change if the `User` interface would have a index of any type, eg:
``` typescript
interface User {
    [_: string]: any;
}
```